bot/submission/submit.py

```diff
@@ -177,6 +188,18 @@ async def post_build(self, build: Build, *, purpose: str, guilds: Sequence[Guild
             message = await channel.send(embed=em)
             await msg.track_message(channel.guild.id, build.id, message.channel.id, message.id, purpose)
 
+            if purpose == "view_pending_build":
+                # Add initial reactions
+                try:
+                    await message.add_reaction(APPROVE_EMOJIS[0])
```

Comment:
Btw we really should wait a bit between reactions because apparently discord's reaction add API limit is quite strict

Reply:
I don't see anything specific to reactions in the [docs](https://discord.com/developers/docs/topics/rate-limits), we should be safe with the 50/s rate but if you are paranoid about it I'll add delays

Reply:
The limit is undocumented and got reduced recently. I know because Starboard was affected. We should generally be fine at this scale, but it doesn't hurt to add a small delay.